pkg/types/types.go

```diff
@@ -71,13 +71,14 @@ var SupportedDevices = map[DeviceType]int{
 	AcceleratorType: 0x12,
 }
 
-// ResourceConfig contains configuration paremeters for a resource pool
+// ResourceConfig contains configuration parameters for a resource pool
 type ResourceConfig struct {
-	ResourcePrefix string           `json:"resourcePrefix,omitempty"` // optional resource prefix that will ovewrite global prefix specified in cli params
-	ResourceName   string           `json:"resourceName"`             // the resource name will be added with resource prefix in K8s api
-	DeviceType     DeviceType       `json:"deviceType,omitempty"`
-	Selectors      *json.RawMessage `json:"selectors,omitempty"`
-	SelectorObj    interface{}
+	ResourcePrefix string `json:"resourcePrefix,omitempty"` // optional resource prefix that will overwrite
```

Comment:
nit: you can put long comments above the variable, I think its clearer that way, especially for multiline comments

Reply:
done